REQUIREMENTS/link-relation-types.md

```diff
@@ -44,13 +44,15 @@ when you think about how you model things in UML: inheritance, implementation, a
 - "Previous": version of the document
 - "member", "Item"
 - "manifest": point to SW implementing the TD
+- "alternate": point to other rappresenatations of the Web Thing. (i.e. HTML document, RDF ...)
 - "Profile": denote compliance
```

Comment:
"profile" -> lower case also

Reply:
yeah, I found the notation a bit strange. Plus some of them are not even IANA, but reading the text it seems they are (there is a dedicated section for NO-IANA proposal). I think the document should be reviewed, I can do that in this PR. Just noticed that I did not modify what you're comment is pointing to, it was there before my changes.

Reply:
Even though I know that @relu91 did not add the initial capitalized terms but I think we can change them in this PR ..